aleph-client/src/lib.rs

```diff
@@ -43,25 +43,10 @@ pub mod utility;
 /// Waiting for some events API.
 pub mod waiting;
 
+pub use ::primitives::*;
```

Comment:
I usually don't like star imports, but it might actually make sense here. What probbably doesn't though is the leading `::`, no?

Reply:
Actually here rust requires leading ::, because it could be ambiguous...